src/playground/service.clj

```diff
@@ -71,7 +75,9 @@
     ["/api" :get (into component-interceptors [http/json-body (param-spec-interceptor ::api :query-params) `api])]
     ["/invoices/insert" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.insert/api :query-params) `invoices.insert/perform])]
     ["/invoices/:id" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.retrieve/api :path-params) `invoices.retrieve/perform])]
-    ["/invoices/delete" :get (into component-interceptors [http/json-body `invoices.delete/perform])]})
+    ["/invoices/delete" :get (into component-interceptors [http/json-body `invoices.delete/perform])]
+    ;; ["/nests/:species" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.retrieve/api :path-params) `nests.retrieve/perform])]
```

Comment:
similar route to "/invoices/:id" to query the `nests` table WIP

Reply:
Don't add WIPs though

Remember: tackle one problem at a time, minimizing noise for the code reviewer